ena-submission/scripts/test_ena_submission_integration.py

```diff
@@ -383,56 +386,49 @@ def get_revisions(modify_manifest: bool = False):
         return revised_sequences
 
 
-def mock_requests_post():
-    mock_response = mock.Mock()
+def mock_requests_post() -> Mock:
+    mock_response = Mock()
     mock_response.status_code = 204
     mock_response.ok = True
     return mock_response
 
 
-def get_dummy_group():
-    return [
-        {
-            "group": {
-                "institution": "University of Test",
-                "address": {"city": "test city", "country": "Switzerland"},
-                "groupName": "test group",
-            }
+TEST_GROUP = [
```

Comment:
```suggestion
TEST_GROUP: Final = [
```

Reply:
Nice, o3 tells me it's not redundant despite the upper case implying a constant!